packages/core/admin/admin/src/utils/uniqueAdminHash.js

```diff
@@ -17,8 +17,6 @@ export default async function hashAdminUserEmail(payload) {
   try {
     return await utils.digestMessage(payload.email);
   } catch (error) {
-    const hash = await import('hash.js');
-
-    return hash.sha256().update(payload.email).digest('hex');
+    return '';
```

Comment:
Do you handle this case somewhere else? I'd be afraid we end up with a few empty values in Amplitude without knowing why.

Reply:
Not really, but I also don't see a viable way to handle this other way. Don't know why it would, but in case subtle crypto fails, there is no other way to hash user email on the admin.  I could add an obvious string, something like `anonymousAdminUser` so we could quickly see on Amplitude in case this specific thing is failing. But on the other hand if we start seeing non-anonymous admin events without a user id on Amplitude, this would be the only place to look for.

Reply:
Could we not send the tracking event, if the hash can not be generated? I don't know if that makes sense and leave it up to you :) I was just wondering if it would be good to handle this case for the sake of data-consistency.

Reply:
I see your point, but I also need to add some context for reassurance. It's better to send the event regardless in my opinion, because even without the user id available (in case that happens), there's going to be a `deviceId` available on the data received by amplitude. That would allow it to map the events to users retrospectively if we need to push some sort of fix for the matter. But I don't expect us needing to do it in the first place